TypeCobol.Test/Parser/Preprocessor/DirectiveTestFiles/ExecSqlInclude.cbl

```diff
@@ -19,3 +19,18 @@
 000007 exec sql include textname6 endexec.
 000008 display "ok ?".
 000009 exec sql delete * from table end-exec. 
+
+000001 exec sql include textname7
```

Comment:
Can you create a test which execute all step till CrossCheck ?

Reply:
Done. See ea5ad60b085af91a8fbad1cf92ee44e46fa51cbe.